src/@orb.yml

```diff
@@ -1,12 +1,7 @@
 version: 2.1
-
-description: >
+description: |
   SauceCtl orb allow you to manage simply the task to run with saucectl.
 
-  display:
+display:
   home_url: "https://www.saucelabs.com/"
```

Comment:
@FriggaHel os the home_url a link to the organization or to the runner? just wondering if we want to link to the test-runner toolkit docs.

Reply:
> Use the `display` key to add clickable links to the orb registry for both your `home_url` (the home of the product or service), and `source_url` (the git repository URL).
https://circleci.com/docs/2.0/orb-author/#orbyml

You're right @vrunoa , maybe this is better to redirect to test-runner toolkit or sauce-docs directly ?

Reply:
my preference would be to redirect to the test-runner docs.

Reply:
Then we're good to go :)